### DISCUSSION TITLE
How to invert report order ?

### DISCUSSION THREAD

**Discussion by MathRobin** [vitest-dev/vitest#6778] [Q&A]
Currently, when i'm doing some tests, i have this kind of reports. Firstly the count of tests passing,  failing, ... and after follows the coverage.

But when my CLI runs the test, the console is not enough large in height to show me the count of passing, failing, ...

My question is simple : is it possible to configure somewhere this order ? Any way to do it ? I understand that it's not a problem for some of you, but for me it's really boring and not practical. Does it need a flag somewhere from me ? A whole complete new custom reporter ?I don't know from where to start

![2024-10-23_15-21](https://github.com/user-attachments/assets/5a989e75-0e44-4095-ae1a-2d95979a6c73)

**Comment by AriPerkkio** [comment]
I don't think there is a way. Vitest calls test reporter's `onFinished` before coverage providers are called:

https://github.com/vitest-dev/vitest/blob/5df7414bf6e66ed379660e276cfb7746980394d2/packages/vitest/src/node/core.ts#L618-L619
